AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToHearingConfirmationNotificationTests.cs

```diff
@@ -162,6 +162,48 @@ public void should_map_to_joh_confirmation_notification()
             result.Parameters.Should().BeEquivalentTo(expectedParameters);
         }
 
+        [Test]
+        public void should_map_to_ejud_joh_confirmation_notification()
+        {
+            var expectedNotificationType = NotificationType.HearingConfirmationEJudJoh;
+            var participant = new ParticipantResponse
+            {
+                Id = Guid.NewGuid(),
+                Username = "testusername@hmcts.net",
+                CaseRoleName = "caserolename",
+                ContactEmail = "contact_judiciary@hmcts.net",
```

Comment:
Should this not fail because it is not an EJud email? Is `hearing.IsJudicialOfficeHolderEJud()` mocked out?